snuba/web/rpc/v1/resolvers/R_eap_spans/common/sampling_in_storage_util.py

```diff
@@ -68,7 +74,14 @@ def _get_bytes_scanned_limit() -> int:
 
 
 def _get_query_bytes_scanned(res: QueryResult) -> int:
-    return cast(int, res.result.get("profile", {}).get("progress_bytes", 0))  # type: ignore
+    profile_dict = res.result.get("profile")
+    if profile_dict is None:
+        raise QueryResultMissingInfoException("profile", res)
```

Comment:
this will fail a query for a user, don't do this, log it as an error and sentry will pick it up

Reply:
you could even add the result to the span (minus the `data` key which you should not include